client/src/App.js

```diff
@@ -8,11 +9,14 @@ import VideoPlayer from "./Components/VideoPlayer";
 function App() {
   const [cards, setCards] = useState([]);
   const [open, setOpen] = useState(false);
+  const [searching, setSearching] = useState(false);
   const [watchVideo, setWatchVideo] = useState(false);
   const videoLink = useRef(null);
 
   const onClose = () => setOpen(undefined);
 
+  const stopSearching = () => setSearching(undefined);
```

Comment:
initial value for `searching` state is `false`. 
please set `false` here as well.